app/src/main/java/ch/epfl/sdp/artificial_intelligence/RandomEnemyGenerator.java

```diff
@@ -0,0 +1,77 @@
+package ch.epfl.sdp.artificial_intelligence;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Random;
+import java.util.Timer;
+
+import ch.epfl.sdp.Enemy;
+import ch.epfl.sdp.GeoPoint;
+import ch.epfl.sdp.Player;
+
+public class RandomEnemyGenerator extends EnemyGenerator {
```

Comment:
The enemy class will change a lot, the generator will need to be adapted.

Reply:
Yes, that's why we did not wanted to go too deeply (and we did not had a lot of time due to the misunderstanding)